WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.m

```diff
@@ -305,14 +248,6 @@ - (void)configureSearchWrapper
     self.searchWrapperView.backgroundColor = [WPStyleGuide wordPressBlue];
 }
 
-- (NSDictionary *)propertiesForAnalytics
```

Comment:
I think we still need this guy ;)  
I can't find the commit where it was removed, so maybe due to a wonky merge?